frontend/src/components/managePermissions/UserDataTable.vue

```diff
@@ -79,7 +80,7 @@ function deleteAssignment(assignment: FamApplicationUserRoleAssignmentGet) {
         acceptLabel: 'Remove',
         acceptClass: 'p-button-danger',
         accept: () => {
-            emit('deleteUserRoleAssignment', assignment);
+            emits('deleteUserRoleAssignment', assignment);
```

Comment:
Maybe lets just name it "emit" (no 's'), just to be consistent:
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/68a7f752-b0d4-43c7-ba9f-8afab207cbbc)

Reply:
just did. Thanks, Ian!